remote-query/remote-query-client/src/main/resources/org/infinispan/query/remote/client/query.proto

```diff
@@ -13,22 +15,20 @@ message QueryRequest {
     */
    required string queryString = 1;
 
-   // NOTE: id 2 was used for sort criteria, which is now into the ORDER BY clause inside the query string
```

Comment:
Coamptaibility with 9.x series is already lost due to TypeId change so we might as well remove some old stuff

Reply:
nope. we keep it.